#!/usr/bin/env python
PACKAGE = "modelos_cinematicos"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("Position_X",    double_t,    0, "Valor de X en el plano", 0.2,  -0.36, 0.36)
gen.add("Position_Z",    double_t,    0, "Valor de Z en el plano", 0.25,  -0.1,0.36)
gen.add("LOWER_elbow", bool_t , 0, "Orientaciòn deseada de codo",True)
gen.add("mov_inicial", bool_t , 0, "Marcar para enviar a la posición inicial",False)
gen.add("Ready", bool_t , 0, "Marcar para habilitar movimiento en simulaciòn",False)
gen.add("Dynamixel", bool_t , 0, "Marcar para habilitar movimiento robot real",False)

exit(gen.generate(PACKAGE, "InverseKinematics_2dof", "IK_2G"))
